src/cms/views/pages/page_actions.py

```diff
@@ -523,6 +523,7 @@ def get_pages_list_ajax(request):
                 ),
             }
         )
+    result = sorted(result, key=lambda k: k['name'])
```

Comment:
```suggestion
    result = sorted(result, key=lambda k: k["name"])
```
Our coding style requires double quotes. If you install the cms with the `./dev-tools/install.sh` script or execute `pipenv run pre-commit install`, you don't have to bother about this and can just use our pre-commit hook to format your changed files.

Reply:
Easier said than done on my machine :sweat_smile: